src/test/groovy/com/github/vkorobkov/jfixtures/JFixturesNullTest.groovy

```diff
@@ -0,0 +1,107 @@
+package com.github.vkorobkov.jfixtures
```

Comment:
Let's have a separate sub packages for testing how different yaml types translate to SQL? Something like `com.github.vkorobkov.jfixtures.integration.types`